src/WasmDis.ts

```diff
@@ -359,9 +359,11 @@ export enum LabelMode {
 
 export interface IDisassemblerResult {
   lines: Array<string>;
-  offsets?: Array<number>
+  offsets?: Array<number>;
   done: boolean;
+  bodyFunctionOffsets?: Array<Array<number>>;
```

Comment:
Nit: "body function" is grammatically weird to me. I'd prefer all instances changed to "function body"

Reply:
this is done